Inc/HALAL/Services/Flash/Flash.hpp

```diff
@@ -0,0 +1,81 @@
+/*
+ * Flash.hpp
+ *
+ *  Created on: 19 oct. 2022
+ *      Author: Pablo
+ */
+
+#ifndef HALAL_SERVICES_FLASH_FLASH_HPP_
+#define HALAL_SERVICES_FLASH_FLASH_HPP_
+
+#include "stm32h7xx_hal.h"
+#include "../../../C++Utilities/CppUtils.hpp"
+
+#define FLASHWORD 8
+#define FLASH_START_ADDRESS 0x08000000
+#define FLASH_SECTOR0_START_ADDRESS 0x08000000
+#define FLASH_SECTOR1_START_ADDRESS 0x08020000
+#define FLASH_SECTOR2_START_ADDRESS 0x08040000
+#define FLASH_SECTOR3_START_ADDRESS 0x08060000
+#define FLASH_SECTOR4_START_ADDRESS 0x08080000
+#define FLASH_SECTOR5_START_ADDRESS 0x080A0000
+#define FLASH_SECTOR6_START_ADDRESS 0x080C0000
+#define FLASH_SECTOR7_START_ADDRESS 0x080E0000
+#define FLASH_END_ADDRESS 0x080FFFFF
+#define SECTOR_SIZE_IN_WORDS 32768
+
+/*
```

Comment:
Cambiar comentario a inglés, te lo paso yo si quieres.

Reply:
Viéndolo mejor, y debido a que va a ser un proyecto open source, creo que deberiamos hacer ya un README, pasar esto al README, y incluir un comentario que diga:
_IMPORTANT: Study Flash README before using this service._

Reply:
Este tema es para una issue.